### ISSUE TITLE
Bye Bye Angular

### ISSUE THREAD

**Issue by lin-d-hop** [openfoodfoundation/openfoodnetwork#9477]
## What is the problem we are solving 

This is an epic to capture issues of removing Angular from the OFN codebase. 
We are replacing AngularJS with StimulusReflex

## Useful information for inception

https://community.openfoodnetwork.org/t/reactive-rails

Slack Channel - #bye-bye-angular

**Comment by rioug** [comment]
Karma is now marked as deprecated : https://github.com/karma-runner/karma?tab=readme-ov-file#karma-is-deprecated-and-is-not-accepting-new-features-or-general-bug-fixes

**Comment by dacook** [comment]
Noting that there is more Angular that's not yet raised in sub-issues, eg:
- shopfront
- ~~admin order cycles~~ (see next comment)
- ~~inventory~~ (but we don't plan to migrate this page, instead it will be removed in the future)

**Comment by RachL** [comment]
@dacook FYI when @chahmedejaz estimated the work on the new admin OC pages, we've also decided that we would not migrate and start fresh.

Regarding the shopfront, is it all tabs or only the product list?

**Comment by dacook** [comment]
I'll have a look at the shopfront now. I think this is everywhere that angular is used that is customer-facing:
* mobile menu
* currently selected shop name
* cart button and sidebar
* homepage "brand story" (red expanding area)
* shops list
* shop page:
  * order cycle selector
  * products list
  * about image
  * producers list
* my account
  * edit order
  * credit cards
  * transactions
* map page
* producers page
* groups page

I haven't looked closely, but I don't think it's used during checkout at all.

**Comment by RachL** [comment]
> I haven't looked closely, but I don't think it's used during checkout at all.

@dacook indeed, we have removed Angular when introducing 3 steps checkout as required by EU legislation.

**Comment by rioug** [comment]
The checkout pages are using turbo, but some element around the page are still using angular, the cart for instance.

**Comment by dacook** [comment]
I've added sub-issues for all shopfront areas that use Angular. Some of them I've requested Rachel to review and add any relevant context.
Most of them will require some planning and quite a bit of re-implementation.
